### DISCUSSION TITLE
部分区域重影严重 求解决方案

### DISCUSSION THREAD

**Discussion by Sostay** [F9y4ng/GreasyFork-Scripts#371] [问答专区(Question / Answer)] [answered]
Github的Code区域 ——这个可以通过关闭阴影解决

![image](https://github.com/user-attachments/assets/388c0859-ef88-418b-8592-1fa67df2c067)

ComfyUI的canvas标签——这个不管怎么修改都没用 之前版本的脚本没有出现过这个问题

![image](https://github.com/user-attachments/assets/cc39812b-c112-409f-8557-6ab0180162e6)

**Comment by F9y4ng** [comment, marked as answer]
* Github code 参考 #266 

* ComfyUI canvas 是由于fillText()方法阴影偏移量未还原初始定义造成的，这个问题会在下个月更新中发布修复。这是9月份月更新中新增的实验性功能（对canvas元素提供字体渲染）引出的新bug。该站点的 Canvas 元素`CanvasRenderingContext2D`属性有定义新的偏移量（浏览器默认为0），所以会出现你遇到的问题，可以临时排除渲染该站点来避免此问题发生。

**针对性修复后的效果：**
![screenshot_2024-09-10_00-08-58](https://github.com/user-attachments/assets/c1655691-e419-46c0-a30d-e4e831b678be)

**Comment by F9y4ng** [comment]
请及时更新至 v2024.09.11.1 。

**Reply by Sostay** [reply]
可以了 非常感谢！

**Reply by Sostay** [reply]
> 请及时更新至 v2024.09.11.1 。

现在飞书项目表格也出现类似的问题了，之前是没有的 目前使用版本 2024.10.05.1
![image](https://github.com/user-attachments/assets/3b717921-2902-48f7-ac5e-8206bf67a5b6)

**Reply by F9y4ng** [reply]
按 #145 试试，如果不行，你得提供能够匿名访问的地址或提供测试账户。
